pg_mooncake/results/c6a.4xlarge.json

```diff
@@ -0,0 +1,58 @@
+{
+    "system": "PostgreSQL with pg_mooncake",
+    "date": "2025-01-08",
+    "machine": "c6a.4xlarge, 500gb gp2",
+    "cluster_size": 1,
+    "comment": "",
+
+    "tags": ["C", "columnstore-oriented", "PostgreSQL compatible"],
```

Comment:
No existing database in ClickBench is tagged as `columnstore-oriented`, but many existing databases are tagged as `column-oriented`.

Let's rename that tag.